pandas/core/groupby.py

```diff
@@ -2315,6 +2315,11 @@ def __init__(self, index, grouper=None, obj=None, name=None, level=None,
                 # groupby
                 else:
                     cat = self.grouper.unique()
```

Comment:
yep that is fine (though return a new object, *never* use ``inplace``).

add the method to a ``CI`` as well (which just calls it on the ``.values``), its a pass thru